<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="box">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Panel</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="0,0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item row="1" column="0" rowspan="2">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Generacja</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Rozkład:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lab1">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Średnia:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QSpinBox" name="point_num">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="generation_btn">
                <property name="text">
                 <string>Generuj</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="sort_btn">
                <property name="text">
                 <string>Sortuj</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QSpinBox" name="criterium_num">
                <property name="minimum">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_4">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Liczba obiektów:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Kryterium:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="distribution_select">
                <item>
                 <property name="text">
                  <string>Eksponencjalny</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Jednostajny</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gaussa</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Poissona</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QLabel" name="lab2">
                <property name="text">
                 <string>Odchylenie:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QDoubleSpinBox" name="odch_num">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QDoubleSpinBox" name="mean_num">
                <property name="minimum">
                 <double>-99999999999999997168788049560464200849936328366177157906432.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999999999999999993220948674361627976461708441944064.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Edytor kryteriów</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QPushButton" name="addCrit_btn">
                <property name="text">
                 <string>Dodaj</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="deleteCrit_btn">
                <property name="text">
                 <string>Usuń</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QTableWidget" name="criteriaTable">
                <property name="autoScrollMargin">
                 <number>16</number>
                </property>
                <property name="rowCount">
                 <number>1</number>
                </property>
                <property name="columnCount">
                 <number>2</number>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <row/>
                <column/>
                <column/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>140</height>
              </size>
             </property>
             <property name="title">
              <string>Akcje</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="0" colspan="2">
               <widget class="QPushButton" name="animation_btn">
                <property name="text">
                 <string>Renderuj animację</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Algorytm:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4" colspan="2">
               <widget class="QPushButton" name="start_btn">
                <property name="text">
                 <string>Rozwiąż</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="stop_btn">
                <property name="text">
                 <string>Przerwij</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QPushButton" name="benchmark_btn">
                <property name="text">
                 <string>Benchmark</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="algorithm_select">
                <item>
                 <property name="text">
                  <string>Naiwny bez filtracji</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Naiwny z filtracją</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Oparty o punkt idealny</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2" colspan="2">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Edytor wartości</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="0">
               <widget class="QPushButton" name="addVal_btn">
                <property name="text">
                 <string>Dodaj</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="load_btn">
                <property name="text">
                 <string>Wczytaj z pliku</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="deleteVal_btn">
                <property name="text">
                 <string>Usuń</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="save_btn">
                <property name="text">
                 <string>Zapisz</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="4">
               <widget class="QTableWidget" name="valuesTable"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Wyniki</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="PlotWidget" name="graph"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>plotwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
